tests/python/pants_test/pantsd/pantsd_integration_test_base.py

```diff
@@ -81,6 +82,14 @@ def assert_stopped(self):
         return self._pid
 
 
+@dataclass(frozen=True)
+class PantsdRunContext:
+    runner: Callable[..., Any]
```

Comment:
FYI `Callable` will do the same thing. This is the more explicit version of it. I think the more explicit is better here because it signals "I know this is the most precise we'll ever want to get."